src/spec/integration/subscriptions.spec.coffee

```diff
@@ -0,0 +1,71 @@
+{SphereClient} = require '../../lib/main'
+Config = require('../../config').config
+testConfig = require('../../test-config').config
+Promise = require 'bluebird'
+debug = require('debug')('spec-integration:products')
+
+newSubscription =
+  key: 'testKey'
+  destination:
+    type: 'IronMQ'
+    uri: "https://#{testConfig.iron_mq_url}/3/projects/#{testConfig.iron_project_id}/queues/test-queue/webhook?oauth=#{testConfig.iron_token}"
+  messages: [
+    resourceTypeId: 'inventory-entry'
+    types: ['InventoryEntryDeleted']
+  ]
+
+describe 'Integration Subscriptions', ->
+
+  beforeEach (done) ->
+    @client = new SphereClient config: Config
+    done()
+
+  afterEach (done) ->
+    debug 'Removing all subscriptions'
+    @client.subscriptions.all().fetch()
+    .then (response) =>
+      Promise.map(response.body.results, (sub) =>
+        @client.subscriptions.byId(sub.id).delete(sub.version)
+      )
+    .then () ->
+      done()
+
+  it 'should create subscription', (done) ->
+    @client.subscriptions.save(newSubscription)
+    .then (response) =>
+      expect(response.statusCode).toBe 201
+      expect(response.body.id).toBeDefined()
+      done()
+  , 20000
+
+  it 'should update message subscriptions', (done) ->
+    @client.subscriptions.save(newSubscription)
+    .then (response) =>
+      @client.subscriptions.byId(response.body.id).update(
```

Comment:
So isn't it better for someone who will need it, to create an issue and a PR afterwards? I think it's a different feature. Doesn't look like it's a rule that when we create a new endpoint, we need to create a syncAction class as well.